packages/rancher-wins-upgrader/package.yaml

```diff
@@ -2,3 +2,4 @@ url: https://github.com/rancher/wins.git
 subdirectory: charts/rancher-wins-upgrader
 commit: 4d46d8dd81cd741a51313923d459f28beea56800
 version: 100.0.1
+doNotRelease: true
```

Comment:
can we remove all the doNotRelease tags in favor of just making the modification to the latest package checked in?

Reply:
updated